dbt_copilot_helper/commands/codebase.py

```diff
@@ -189,11 +191,11 @@ def build(app, codebase, commit):
 def deploy(app, env, codebase, commit):
     """Trigger a CodePipeline pipeline based deployment."""
     application = load_application_with_environment(app, env)
-
     check_codebase_exists(application, codebase)
     check_image_exists(application, codebase, commit)
 
-    codebuild_client = boto3.client("codebuild")
+    session = get_aws_session_or_abort()
```

Comment:
As we have the environment loaded, we can access the session for the environment via `application.environments.get(env).session`